script-library/test/regression/test.sh

```diff
@@ -1,17 +1,47 @@
 #!/usr/bin/env bash
 IMAGE_TO_TEST="${1:-debian}"
+RUN_ONE="${2:-false}" # false or script name
+USE_DEFAULTS="${3:-true}"
+RUN_COMMON_SCRIPT="${4:-true}"
+PLATFORMS="${5:-"linux/amd64"}"
 
-if [ "$IMAGE_TO_TEST" = "centos" ]; then
-    DISTRO="redhat"
-elif [ "${IMAGE_TO_TEST}" = "ubuntu" ]; then
+if [[ "${IMAGE_TO_TEST}" = *"ubuntu"* ]]; then
     DISTRO="debian"
+elif [[ "${IMAGE_TO_TEST}" = *"alpine"* ]]; then
+    DISTRO="alpine"
+elif [[ "$IMAGE_TO_TEST" = *"centos"* ]] || [[ "$IMAGE_TO_TEST" = *"redhat"* ]]; then
+    DISTRO="redhat"
 else
     DISTRO=$IMAGE_TO_TEST
 fi
 
 set -e
 
-cd "$(cd "$(dirname "$0")" && pwd)/../.."
+cd "$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/../.."
 echo -e  "🧪 Testing image $IMAGE_TO_TEST (${DISTRO}-like)..."
-docker build --build-arg DISTRO=$DISTRO --build-arg IMAGE_TO_TEST=$IMAGE_TO_TEST -f test/regression/Dockerfile .
+
+CURRENT_BUILDERS="$(docker buildx ls)"
+if [[ "${CURRENT_BUILDERS}" != *"vscode-dev-containers"* ]]; then
+    docker buildx create --use --name vscode-dev-containers
```

Comment:
this line and the else condition look a bit funny to me: One has a `--use` and one has `use`.  Might be fine, not too familiar with buildx and its args

Reply:
Yeah if it already exists, then you would use the `use` command.  The `create --use` command is like running create, then use, but only works if the builder doesn't exist.